### DISCUSSION TITLE
Did I dream?

### DISCUSSION THREAD

**Discussion by mattbdc** [solidjs/solid#1963] [General]
I remember, perhaps wrongly, there was a solidjs/signals repo about a month ago that was meant to be an incubator for new ideas, this was after a few months of general darkness regarding the march to astro/2.0/better async patterns, I can't find it now? what happened?  

 If I remember rightly this linked from some hackmd.io thoughts? perhaps not intended to be published yet?

**Comment by lxsmnsyc** [comment]
This one? https://github.com/solidjs/signals

**Comment by mattbdc** [comment]
Ah yes, was this hidden for a period of time? And recently restored as part of the incoming new release?  Or maybe PEBKAC

**Reply by lxsmnsyc** [reply]
It was private for a long time, only made public months ago. It was never re-hidden

**Comment by snnsnn** [comment]
So, the answer is yes? Maybe.
